nais/nais.yaml

```diff
@@ -38,12 +38,23 @@ spec:
     instance: kandidat
   tokenx:
     enabled: true
+  azure:
+    application:
+      enabled: true
+      allowAllUsers: false
+      claims:
+        extra:
+          - NAVident
+        groups:
+          - id: {{ ad_gruppe_modia_generell_tilgang }}
+          - id: {{ ad_gruppe_modia_oppfølging }}
```

Comment:
Er disse lagt til i nais-configen for dev og prod?

Reply:
nice catch.. hadde glemt